horovod/mxnet/__init__.py

```diff
@@ -87,3 +93,6 @@ def broadcast_parameters(params, root_rank=0):
     for _, p in params:
         broadcast_(p, root_rank, str(count))
```

Comment:
Nit: let's change tensor name to `'broadcast_parameters.%d' % count` for better debuggability?

Reply:
Ah, right. Nevermind.